src/Entity/JoindInUser.php

```diff
@@ -35,28 +35,25 @@ class JoindInUser implements \JsonSerializable
      * @var string
      */
     private $displayName;
+    /**
+     * @ORM\Column(type="boolean", nullable=false, options={"default":false})
+     *
+     * @var bool
+     */
+    private $organizer;
```

Comment:
maybe rename this to `isOrganizer`, just to make it more readable and intuitive?

Reply:
I'm not 100% sure on saying 'is' in properties ... if you already set it I would be fine with it but changing it doesnt make that much sense ...